client/src/backoffice/InboxPage.tsx

```diff
@@ -1,202 +1,208 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+
+// libraries
 import classNames from 'classnames';
 import { Formik, Form, Field, ErrorMessage, FieldProps, FormikActions } from 'formik';
-import { InboxFormSchema } from '../lib/schemas';
+import { useToasts } from 'react-toast-notifications';
 
 /* Helpers */
 import { getEvents, PoapEvent, sendNotification } from '../api';
 
 /* Components */
 import { SubmitButton } from '../components/SubmitButton';
+import FilterSelect from '../components/FilterSelect';
 
-/* Typings */
-import { Value } from '../types';
+// schema
+import { InboxFormSchema } from '../lib/schemas';
 
-interface IInboxState {
-  events: PoapEvent[];
-  initialValues: IInboxFormValues;
-}
+/* Typings */
+import { Name, NotificationType, RecipientType } from '../types';
 
-interface IInboxFormValues {
+type IInboxFormValues = {
```

Comment:
Since we are using a `type` now we should change its name from `IInboxFormValues` to `InboxFormValues` since the first upper case `I` means Interface.